universal-login-provider/src/ULIFrameProvider.ts

```diff
@@ -99,9 +112,17 @@ export class ULIFrameProvider {
     this.setDashboardVisibility(false);
   }
 
+  private setElementVisibility = (element: HTMLDivElement, isVisible: boolean) => {
```

Comment:
Can be reduced to something like https://github.com/UniLogin/UniLogin/blob/master/universal-login-provider/src/ULIFrameProvider.ts#L74